doc/nrf/releases_and_maturity/release_notes.rst

```diff
@@ -21,6 +20,7 @@ This page is included only in the latest documentation, because it might contain
    :caption: Subpages:
 
    releases/release-notes-changelog
```

Comment:
Please remove the changelog from the TOC, 2.5.0 should be the top entry at the time of the release

Reply:
thanks